docs/proposals/odo-v3-cli/odo-list.md

```diff
@@ -0,0 +1,29 @@
+# `odo list`
+
+list all odo related information from the current namespace
+
+## Flags
+- `--namespace` - list components from the given namespace instead of current namespace.
+
+## example
+```
+$ odo list
```

Comment:
I know we have discussed this before, but I keep circling back to thinking that this is an inconsistent behavior :/

Reply:
> I know we have discussed this before, but I keep circling back to thinking that this is an inconsistent behavior :/

Feel free to suggest better alternative